codalab/worker/worker.py

```diff
@@ -78,6 +78,8 @@ def __init__(
         delete_work_dir_on_exit=False,  # type: bool
         # A flag indicating if the worker will exit if it encounters an exception
         exit_on_exception=False,  # type: bool
+        # mainly to avoid trying to use the docker socket which we may not have permissions to.
+        runtime_docker=True,  # type: bool
```

Comment:
Take this parameter out in this branch (you can add it in another branch if you need though)